.goreleaser.yaml

```diff
@@ -0,0 +1,67 @@
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - id: server
+    main: ./bin/server
+    binary: bytebase
+    ldflags:
+      - -w -s
+      - -X github.com/bytebase/bytebase/bin/server/cmd.version={{.Version}}
+      - -X github.com/bytebase/bytebase/bin/server/cmd.goversion=${GO_VERSION}
+      - -X github.com/bytebase/bytebase/bin/server/cmd.gitcommit={{.Commit}}
+      - -X github.com/bytebase/bytebase/bin/server/cmd.buildtime={{.Timestamp}}
+      - -X github.com/bytebase/bytebase/bin/server/cmd.builduser=goreleaser
+    tags:
+      - release
+    env:
+      - VERSION="development"
+      - GO_VERSION="1.16.5"
+      - GIT_COMMIT="unknown"
+      - BUILD_TIME="unknown"
+      - BUILD_USER="unknown"
+    goos:
+      - linux
+      - darwin
+    goarch:
+      - amd64
+      - arm64
```

Comment:
I'm not quite sure if we support linux/arm64 since its Postgres binary isn't packaged. But I'm pretty sure the Postgres darwin/amd64 binary works for both amd64 and arm64 (my laptop).

We can get a AWS VM to test it.